tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -315,3 +317,20 @@ Replace "Prometheus" With "Grafana" In Rhods-Monitor-Federation
     OpenShiftLibrary.Oc Patch    kind=ServiceMonitor
     ...                   src={"spec":{"selector":{"matchLabels": {"app":"grafana"}}}}
     ...                   name=rhods-monitor-federation   namespace=redhat-ods-monitoring  type=merge
+
+Verify CPU And Memory Requests For Image And Base Cuda Pods
+    [Documentation]     Verifies CPU and Memory Requests for image and base cuda pods
+    [Arguments]   ${namespace}=redhat-ods-applications
+    ${pods} =  Oc Get  kind=Pod  namespace=${namespace}
+    FOR    ${pod}    IN  @{pods}
```

Comment:
could you try fetching only the pods of cuda using the filters in Oc get?

Reply:
But we still need to fetch image pods as well ,separately. Why not fetch all of them at once and filter one after the other?

Reply:
ok